internal/components/skills/inject_test.go

```
@@ -371,3 +374,81 @@ func containsFile(files []string, want string) bool {
 	}
 	return false
 }
+
+// --- PluginInstaller interface check tests ---
+
+// mockPluginInstallerAdapter is a minimal adapter that implements PluginInstaller.
+type mockPluginInstallerAdapter struct {
```

golangci-lint (2.12.2)

[error] type mockPluginInstallerAdapter is unused

(unused)